scripts/hammer_command_tree.py

```diff
@@ -0,0 +1,31 @@
+"""Generate hammer command tree in json format by inspecting every command's
+help.
+
+"""
+import json
+
```

Comment:
Actually this is a recommendation from PEP8, for more information check https://www.python.org/dev/peps/pep-0008/#imports